services-js/registry-certs/client/birth/styling.tsx

```diff
@@ -20,6 +20,18 @@ const FOCUS_STYLE = {
   outlineOffset: 1,
 };
 
+export const BACK_BUTTON_STYLING = css({
```

Comment:
You should be able to use the `lnk` class on a `button` to achieve this.

Reply:
Ah, thanks! I _thought_ I recalled a class for that, but didn’t track it down when I searched through the fleet stylesheets.